src/mysql/create.sql

```diff
@@ -34,6 +33,16 @@ CREATE TABLE Notification (
   REFERENCES User (id)
```

Comment:
the foreign key constraint should be deleted since there is no longer a user_id attribute

Reply:
Good catch! Done.